### PR TITLE
Example of running conftest via godog

### PR DESCRIPTION
**Pull request by iantivey**

### CODE REVIEW

File: service_packs/aks-opa/general/json/kube_dashboard/not_present.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+  "resource_changes": [
+    {
+      "type": "azurerm_kubernetes_cluster",
+      "change": {
+        "after": {
+          "add_on_profile": [
```
Comment (paulm-jones) [inline service_packs/aks-opa/general/json/kube_dashboard/not_present.json:7]: Too many underscores

File: service_packs/aks-opa/general/opa/kube_dashboard/kube_dashboard_test.rego [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+test_enabled_kube_dashboard {
+  deny["kubernetes dashboard should be disabled"]
+  with input as
+    {
+      "resource_changes": [
+        {
+          "type": "azurerm_kubernetes_cluster",
+          "change": {
+            "after": {
+              "add_on_profile": [
```
Comment (paulm-jones) [inline service_packs/aks-opa/general/opa/kube_dashboard/kube_dashboard_test.rego:20]: same here

File: service_packs/aks-opa/general/opa/kube_dashboard/kube_dashboard_test.rego [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+                {
+                  "aci_connector_linux": [],
```
Comment (paulm-jones) [inline service_packs/aks-opa/general/opa/kube_dashboard/kube_dashboard_test.rego:22]: would not include all of the unrelated stuff in a unit test

**Comment by paulm-jones** [comment]
I think I am against this as a concept. I feel that just because we could run conftest from Probr, it doesn’t mean that we should.

We are trying to build a tool that positions itself as a way of performing real tests on infrastructure, a tool that deliberately mistrusts policy and guardrails. It perhaps already has a little bit of an identity crisis (which we are making strides to solidify). For me this takes us away from the design goal.

Happy to be persuaded otherwise.

**Comment by iantivey** [comment]
Perhaps it should be a separate tool.  I think there's value in evaluating/validating OPA policies via BDD.  We should discuss whether that's a job for Probr or another tool (Validatr?), although it would be nice to be able to use Probr core engine across both tools.

**Comment by paulm-jones** [comment]
For me that's another project, not the responsibility of Probr.
If we want to separate our Probr core, fine.
I don't think _everything_ has to be BDD. It boxes us in.

**Comment by eddie-knight** [comment]
Apologies for duplicating this PR- didn't realize it was being opened while I was working on my own version of it. Initially I was hoping to push my changes to the Tivey branch, but the workflow got a bit muddled by user error along the way.
